generate_events.py

```diff
@@ -1606,19 +1656,259 @@ def define_sex_events(es: EventMap):
                           coherent thoughts. Instead, your mind is filled with a pink haze, urging you to just accept
                           the pleasure of being used like a piece of meat.
                           """),
+                   Option(EventsSex.PRONE_BONE, OptionCategory.SUB, #Added by me#
+                          "Submit to getting plowed facing down",
+                          transition_text=f"""
+                          You accept his invasion, each thrust making it harder and harder to pull away and form
+                          coherent thoughts. Your legs weaken and you fall to the floor as your mind is filled with a pink haze,
+                          urging you to just accept the pleasure of being used like a piece of meat.
+                          """), 
                    Option(EventsCum.PULL_OUT_CUM_ON_ASS, OptionCategory.DOM,
                           "Have him pull out and cum on your ass",
                           transition_text=f"""
-                          Your requests sways {THEM}.""",
+                          "Not inside!" you barely blurt out between moans and labored breaths.""",
```

Comment:
"Not inside!" is used 6 times (3 dom transitions, in the success and failed transition texts)
Could vary it up with alternatives like "Outside!"